src/lib/ggpo/platform_linux.cpp

```diff
@@ -7,17 +7,12 @@
 
 #include "platform_linux.h"
 
-struct timespec start = { 0 }
-
-uint32 Platform::GetCurrentTimeMS() {
-    if (start.tv_sec == 0 && start.tv_nsec == 0) {
-        clock_gettime(CLOCK_MONOTONIC, &start);
-        return 0
-    }
-    struct timespec current;
-    clock_gettime(CLOCK_MONOTONIC, &current);
-
-    return ((current.tv_sec - start.tv_sec) * 1000) +
-           ((current.tv_nsec  - start.tv_nsec ) / 1000000) +
+uint32_t Platform::GetCurrentTimeMS() {
```

Comment:
Here's my understanding... GGPO will use this function to measure elapsed time (i.e. subtracting one time from another).  `gettimeofday` will return the time according to the system clock, which might get adjusted based on an update from an NTP time source between the two calls.  This could make it appear as if an operation took a negative amount of time.  See https://blog.habets.se/2010/09/gettimeofday-should-never-be-used-to-measure-time.html for more information.  Thoughts?